frontend/pages/play/[roomcode]/members.vue

```diff
@@ -90,9 +90,77 @@ const isReady = computed(() => {
 function routeToReadyScreen() {
   navigateTo(localRoute("play-roomcode-ready"));
 }
+
+const showWelcomeInfobox = ref(false);
+
+function welcomeInfoboxShown() {
+  showWelcomeInfobox.value = false;
+  if (typeof localStorage !== "undefined") {
+    localStorage.setItem("welcomeInfoboxShown", "true");
+  }
+}
+
+onMounted(() => {
+  if (typeof localStorage !== "undefined") {
+    const localStorageValue = localStorage.getItem("welcomeInfoboxShown");
+    if (localStorageValue === null) {
+      showWelcomeInfobox.value = true;
+    } else {
+      showWelcomeInfobox.value = false;
+    }
+  }
+});
 </script>
 
 <template>
+  <Infobox v-if="showWelcomeInfobox" :progressBarExists="false">
```

Comment:
consider setting the welcome info box as fallback content (`<slot><!--here--></slot>`) in the info box component